app/controllers/posts_controller.rb

```diff
@@ -1,32 +1,35 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] Please add cancan for post authorization. Add this at the beginning `load_and_authorize_resource
`